scripts/yamale_validator.py

```diff
@@ -0,0 +1,18 @@
+# Import Yamale and make a schema object:
```

Comment:
Add `# -*- coding: utf-8 -*-` to the top of the file to avoid encoding issues such as:
```
chris@chris-MS-7B77:~/bugs/robust$ python scripts/yamale_validator.py geometry2/164cfa3/164cfa3.bug 
  File "scripts/yamale_validator.py", line 11
SyntaxError: Non-ASCII character '\xf0' in file scripts/yamale_validator.py on line 11, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

Reply:
https://github.com/robust-rosin/robust/pull/359/commits/ff8ced934862ff5624b8b915558db5c5c6061f08